### ISSUE TITLE
output

### ISSUE THREAD

**Issue by vulnz** [enkomio/Taipan#5]
Hello! can you please make html output or text output?

**Comment by enkomio** [comment]
Hi,

the latest Taipan version (which can be downloaded from https://ci.appveyor.com/api/buildjobs/6kc5944jqd6ooj5p/artifacts/Src%2Fdeploy.zip) save the scan result in a JSON format in order to be post-processed. It also display some information about the scan result to the console.

Is that what you want? If not, could you please provide an example of information that you think can be useful to report? 

Please consider that the open source version is created in order to provide a strong scan engine. Additional features (like advanced reporting) are not part of the open version (but are part of the closed version).

**Comment by vulnz** [comment]
thank you for answer.
i'm asking if you have simple txt output or html one, not json, because it is hard to parse it for now

**Comment by vulnz** [comment]
or console results output without json to make > out.txt

**Comment by enkomio** [comment]
I'll consider your request and evaluate its implementation in the next release. Of course, as for the reason mentioned above, I cannot ensure the implementation of a full detailed report.
